java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -394,4 +394,56 @@ public void functionLoad() {
         assertEquals(libName, regularClient.functionLoad(newCode, true).get());
         // TODO test with FCALL
     }
+
+    @Test
+    @SneakyThrows
+    public void copy() {
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("6.2.0"), "This feature added in redis 6.2.0");
+        // setup
+        String source = "{key}-1" + UUID.randomUUID();
+        String destination = "{key}-2" + UUID.randomUUID();
+        long index1 = 1;
+        long index2 = 2;
+
+        // neither key exists, returns false
+        assertEquals(false, regularClient.copy(source, destination).get());
+        assertEquals(false, regularClient.copy(source, destination, false).get());
+        assertEquals(false, regularClient.copy(source, destination, index1, false).get());
+
+        // source exists, destination does not
+        regularClient.set(source, "one").get();
+        assertEquals(true, regularClient.copy(source, destination, false).get());
+        assertEquals(true, regularClient.copy(source, destination, index1, false).get());
+        assertEquals("one", regularClient.get(destination).get());
+        regularClient.select(1).get();
+        assertEquals("one", regularClient.get(destination).get());
+
+        // new value for source key
+        regularClient.select(0).get();
+        regularClient.set(source, "two").get();
+
+        // no REPLACE, copying to existing key on DB 0&1, non-existing key on DB 2
+        assertEquals(false, regularClient.copy(source, destination).get());
+        assertEquals(false, regularClient.copy(source, destination, false).get());
+        assertEquals(false, regularClient.copy(source, destination, index1, false).get());
+        assertEquals(true, regularClient.copy(source, destination, index2, false).get());
+
+        // new value only gets copied to DB 2
+        assertEquals("one", regularClient.get(destination).get());
+        regularClient.select(1).get();
+        assertEquals("one", regularClient.get(destination).get());
+        regularClient.select(2).get();
+        assertEquals("two", regularClient.get(destination).get());
+
+        // both exists, with REPLACE, when value isn't the same, source always get copied to destination
+        regularClient.select(0).get();
+        assertEquals(true, regularClient.copy(source, destination, true).get());
+        assertEquals(true, regularClient.copy(source, destination, index1, true).get());
+        assertEquals("two", regularClient.get(destination).get());
+        regularClient.select(1).get();
+        assertEquals("two", regularClient.get(destination).get());
+
+        // switching back to db 0
+        regularClient.select(0).get();
```

Comment:
you should either run this in a separate client or make sure this is run even if the test fails.